frontend/src/router/router.jsx

```diff
@@ -7,6 +7,9 @@ import PrivateRoute from './PrivateRoute';
 
 // Pages
 import { Splash } from '../pages/Splash';
+import { AllData } from '../pages/allData';
+import { MetadataByType } from '../pages/MetadataByType';
+import { ViewMetadata } from '../pages/Inspection';
```

Comment:
The component exported from a page should generally have the same name as the page.